client/src/components/ProTx/components/charts/PredictiveFeaturesPlot.js

```diff
@@ -10,6 +10,7 @@ import {
 import './PredictiveFeaturesPlot.css';
 
 function PredictiveFeaturesPlot({
+  mapType,
```

Comment:
is this needed?

Reply:
It isn't used, but the same goes for ObservedFeaturesPlot. I'll make another PR to remove all the unused props that I ended up with after refactoring the instructions component.